services/resource-getter.js

```diff
@@ -17,18 +16,6 @@ function ResourceGetter(model, params) {
             ' does not exist.');
         }
 
-        // NOTICE: Do not use "toJSON" method to prevent issues on models that
-        //         override this method.
-        record = record.get({ plain: true });
```

Comment:
Useless code (2 years old) that is the origin of the issue with Smart Field values computation.